scenarios/telemetry/dotnet/telemetry_consumer/Program.cs

```diff
@@ -1,6 +1,6 @@
 using telemetry_consumer;
 
-//System.Diagnostics.Trace.Listeners.Add(new System.Diagnostics.ConsoleTraceListener());
+System.Diagnostics.Trace.Listeners.Add(new System.Diagnostics.ConsoleTraceListener());
```

Comment:
`ConsoleTraceListener` is `IDisposable`. Do we care about ensuring it gets disposed?

Reply:
It's only used for troubleshooting, and should be commented